ras_party/batch_process/batch_request.py

```diff
@@ -0,0 +1,87 @@
+import json
```

Comment:
I don't think this should be in `/batch_process` when all the other endpoints are in `/views` - lets just have one place for routes to live

Reply:
cool! moved `bach_resquest.py` to `/views`